src/v2/guide/class-and-style.md

```diff
@@ -202,3 +202,5 @@ Starting in 2.3 you can provide an array of multiple (prefixed) values to a styl
 ``` html
 <div :style="{ display: ['-webkit-box', '-ms-flexbox', 'flex'] }">
 ```
+
+This will only render the last value in the array which the browser supports. In this example, it will render `display: flex` for browsers that support the unprefixed version of flexbox.
```

Comment:
[comment](https://github.com/vuejs/vue/blob/dev/src/platforms/web/runtime/modules/style.js#L19) said:
`Set them one by one, and the browser will only set those it can recognize`

Reply:
While I find the present explanation super confusing, the one you just gave is perfectly clear @paul-v-m 
I think it should be uptaded to: 
`color: ['red', 'yellow', 'fake(value)'] `
it will only set yellow

Reply:
Because, if I'm understanding it correctly, even if the browser supported prefixed versions of flexbox, `display: flex` would still be rendered, making the example ambiguous.